Tests/SonarQube.TeamBuild.PreProcessor.Tests/RoslynAnalyzerProviderTests.cs

```diff
@@ -61,7 +61,7 @@ public void RoslynConfig_ProjectNotInProfile()
             RoslynAnalyzerProvider testSubject = CreateTestSubject(logger);
 
             // Act
-            AnalyzerSettings actualSettings = testSubject.SetupAnalyzers(mockServer, settings, "unknown.project");
+            AnalyzerSettings actualSettings = testSubject.SetupAnalyzers(mockServer, settings, "unknown.project", null);
 
```

Comment:
Need to test the "known project / unknown branch" case and the "known project / non-default branch" case.